docs/source/standard/auction.rst

```diff
@@ -117,23 +191,31 @@ Schema
    * `valueAddedTaxIncluded` should either be absent or match `Auction.value.valueAddedTaxIncluded`
 
 :awards:
-    List of :ref:`award` objects
+   Array of :ref:`award` objects
 
-    All qualifications (disqualifications and awards).
+   All qualifications (disqualifications and awards).
 
 :contracts:
-    List of :ref:`Contract` objects
+   Array of :ref:`Contract` objects
+
+   |ocdsDescription|
+   Information on contracts signed as part of a process.
+
+:rectificationPeriod:
+   :ref:`period`, auto-generated, read-only
+
+   The period when you can edit the procedure. Editing is possible within 48 hours from the inception of the procedure.
```

Comment:
Plz consider the review here https://github.com/openprocurement/openprocurement.auctions.insider/pull/95